svgdigitizer/svgdigitizer.py

```diff
@@ -157,29 +149,48 @@ def get_real_values(self, xpathdata):
         ynorm = self.trafo['y'](xpathdata[:, 1])
         return np.array([xnorm, ynorm])
 
+    @cached_property
+    def labeled_paths(self):
+        r"""
+        Return all paths which are grouped/label with text elements, warn if
+        unlabeled or ambiguous paths exist.
+        """
+        labeled_paths = {}
+        texts = self.doc.getElementsByTagName('text')
+        for text in texts:
+
+            group = text.parentNode
```

Comment:
Can we check that this is actually a group?

Reply:
I think the parent of a text is always a group since layers are also groups.

Reply:
I think this is not true for top-level texts.

Reply:
It works without a layer. In this case, top_level is 'svg'.
Top-level texts are excluded in a later commit.